.github/automation/performance/benchdnn_comparison.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+
+# *******************************************************************************
+# Copyright 2025 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+
+import sys
+import os
+from collections import defaultdict
+from scipy.stats import ttest_ind
+
+
+def compare_two_benchdnn(file1, file2, tolerance=0.05):
+    """
+    Compare two benchdnn output files
+    """
+    with open(file1) as f:
+        r1 = f.readlines()
+
+    with open(file2) as f:
+        r2 = f.readlines()
+
+    # Trim non-formatted lines and split the prolem from time
+    r1 = [x.split(",") for x in r1 if x[0:8] == "--mode=P"]
+    r2 = [x.split(",") for x in r2 if x[0:8] == "--mode=P"]
+
+    if len(r1) != len(r2):
+        raise Exception("The number of benchdnn runs do not match")
+
+    # Convert to dict and trim \n
```

Comment:
This comment seems misleading. You are just intialising new defaultdicts right? Where is the stripping of the newline.